main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"./migraph"
```

Comment:
I'm getting `can't load package: /home/kv/projects/kinvolk/GOPATH/src/github.com/kinvolk/mountinfo-graph/main.go:4:2: local import "./migraph" in non-local package`, so I think you should replace it with `"github.com/kinvolk/mountinfo-graph/migraph"`.